src/WaipuData.cpp

```diff
@@ -2048,4 +2053,26 @@ PVR_ERROR WaipuData::OnSystemWake()
   return PVR_ERROR_NO_ERROR;
 }
 
+PVR_ERROR WaipuData::GetRecordingLastPlayedPosition(const kodi::addon::PVRRecording& recording,
+                                                    int& position)
+{
+  return PVR_ERROR_NOT_IMPLEMENTED;
```

Comment:
Do we want to implement it in this PR or in a follow-up pr in future?

I noticed that the recordings API `application/vnd.waipu.recordings-v4+json` returns the values `positionPercentage` and `durationSeconds` - maybe we need to calculate the position? Or do you have another idea?

```json
    {
        "id": "1034850919",
        "programId": "772b58e3-f87e-411c-bfc0-f48e41db548f",
        "stationId": "zdfneo",
        "recordingStartTime": "2024-05-30T22:30:41+0200",
        "title": "Zurück in die Zukunft II",
        "epgStartTime": "2024-05-30T22:00:00+0200",
        "genreDisplayName": "Filme",
        "previewImage": "https://images.wpstr.tv/dmb/zdfneo/1897767360/314941785/zno_240530_2200_f8e0833c_zurueck_in_die_zukunft_ii.jpg?res=${resolution}",
        "stationDisplay": "ZDFneo",
        "positionPercentage": 0,
        "durationSeconds": 4962,
        "fullyWatched": false,
        "partiallyWatched": false,
        "isNew": true,
        "locked": false,
        "status": "FINISHED"
    },
```

Reply:
I tried it: It sets the position on backend (as expected) but Kodi also keeps the position locally stored. I didn't expect this, but that`s great. I think we can merge it as it is (and extend it later). Thanks a lot!